doc/incentives/simulation-and-test-scenarios.md

```diff
@@ -1,87 +1,75 @@
-a change in the code will not break integrity or correctness of existing code and which should
-run on every change of the code. They are not useful much for experimentation,
-nor for integration, as they should be focused on running fast to not extend too much
-the complete go unit suite. 
+Simulations run as part of the go unit testing suite. They spawn a configurable number of nodes in a single process and then can run individual interactions. They are mostly useful to verify that a change in the code does not break the integrity or correctness of the system, and thus should be run every time the code changes. They are not of much use for experimentation nor for integration, as they should be focused on running quickly so as not to significantly extend the running time of the complete test suite.
 
 ### Accounting correctness
-This is a test well suited for a simulation. At no point after a change the correctness of accounting should be broken.
-**Description**: Launch x nodes, upload a file, sync, then download from different nodes, and check balances.
-**Expectation**: All balances should be symmetric with each other (`balanceA == -balanceB && balanceB == -balanceA`)
-**Restriction**: Keep balances below threshold so to not emit cheques 
+This is a test well suited for a simulation. At no point after a change should the correctness of accounting be broken.
+
+**Description**: Launch _x_ nodes, upload a file, sync, then download from different nodes, and check balances.  
+**Expectation**: All balances should be symmetric with each other (`balanceA == -balanceB && balanceB == -balanceA`) and at least _some_ of them should be non-zero.  
+**Restriction**: All nodes should have SWAP enabled and a valid chequebook smart contract deployed. All thresholds should be equal and all nodes must be on the same network. Balances should be kept below the payment threshold so no cheques are emitted.  
 
 ### Cheques correctness
-Trigger multiple thresholds crossing (and thus multiple cheques emission).
-**Description**: Launch x nodes, upload a file, sync, then download from different nodes. 
-**Expectation**: An expected set of nodes has (multiple) cheques. The cumulative amount of cheque value and balances match.
-**Restriction**: Fixed (non random) data and size, use snapshot.
+Trigger multiple threshold crossings (and thus multiple cheque emissions).
 
+**Description**: Launch _x_ nodes, upload a file, sync, then download from different nodes.  
+**Expectation**: An expected set of nodes has (multiple) cheques. The cumulative amount of cheque values and balances match. For every pair of nodes _A_ & _B_ the last received cheque to node _B_ from node _A_ should match the last sent cheque from node _A_ to node _B_.  
+**Restriction**: All nodes should have SWAP enabled and a valid chequebook smart contract deployed. File size should be big enough to trigger cheque emissions, so fixed (non random) data and size should be used. Use snapshot.  
```

Comment:
how would snapshots be used here?

Reply:
This does not apply anymore after the experiences with #1737. Will adjust.